### ISSUE TITLE
Base rule extension: [rule-name] <a short description of my proposal>

### ISSUE THREAD

**Issue by mohamedniyaz0506** [typescript-eslint/typescript-eslint#12692]
### Before You File a Proposal Please Confirm You Have Done The Following...

- [x] I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3A%22enhancement%3A+new+base+rule+extension%22) and found none that match my proposal.
- [x] I have searched the [current extension rule list](https://typescript-eslint.io/rules/?=extension#rules) and found no rules that match my proposal.
- [x] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.
- [x] If I used AI assistance to prepare this issue, I have reviewed and followed the [AI Contribution Policy](https://typescript-eslint.io/contributing/ai-policy/).

### My proposal is suitable for this project

- [x] My proposal specifically checks TypeScript syntax, or it proposes a check that requires type information to be accurate.
- [x] I believe my proposal would be useful to the broader TypeScript community (meaning it is not a niche proposal).

### Link to the base rule

Nothing 

### Description

Commute other people for no-leak security  and analysis activities on user interface 

### Fail

```TypeScript
var replace = 'me';
Apply=communication ';
Reprint & chat active ';
```

### Pass

```TypeScript
const replace = 'me';
Apply=communication ';
Reprint & chat active ';
```
